sympy/core/power.py

```diff
@@ -530,6 +530,9 @@ def _eval_is_extended_positive(self):
                 return log(self.base).is_imaginary
 
     def _eval_is_extended_negative(self):
+        if self.exp is S(1)/2:
```

Comment:
`is` works here because 1/2 is singletonized, but it's generally better to be explicit about that and use S.Half if you are using `is` (otherwise use `==`).

Reply:
We should be able to generalize this.

Reply:
Okay I would change that and keep it correct next time. Should I commit this to same branch?